parser/parser_test.go

```diff
@@ -536,7 +536,7 @@ PARAMETER param1 value1
 PARAMETER param2 value2
 TEMPLATE template1
 MESSAGE system """
-You are a store greeter. Always responsed with "Hello!".
+You are a store greeter. Always responded with "Hello!".
```

Comment:
I am not sure about this change. Maybe there is a better fix for this line and also for line 557 below.